user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+
+|
+
+.. image:: ./img/roles/grid_share.png
+
+|
+
+ 
+.. _admin-capabilities-system-info:
+
+System Information
+------------------
+
+If this capability is set to *"System"*, the user can view the system information page. This page contains the list of 
+Oro packages and third-party packages that are installed, and is usually only used by system administrators and 
+integrators.
+
+.. _admin-capabilities-system-config:
+
+System Configuration
+--------------------
+
+If this capability is set to *"System"*, the user can access the :ref:`system configuration page <admin-configuration>`
+to localize the system, change the display and tracking settings, and otherwise change the system configuration.
+
+
+.. _admin-capabilities-tags:
+
+Tag Assign/Unassign
+-------------------
+
+If this capability is set to *"System"*, the user can  assign/unassign :ref:`tags <user-guide-tags>` which are 
+non-hierarchical keywords or phrases assigned to records. They provide additional information about records and
+are visible to all the system users. 
+
+Tags can be successfully utilized by all the users.
+
+
+.. _admin-capabilities-tags-all:
+
+Unassign All Tags From Entities
+-------------------------------
+
+This capability is only meaningful if *"Tag assign/unassign"* is set to *"System"*.
+
+If the both *"Tag assign/unassign"* and *"Unassign All Tags From Entities"* capabilities are set to *"System"*, the 
+users can unassign not only the tags that they have added, but tags added to a record by other users.
+
+This way, you can restrict users from deleting tags made by other users. This is usually available to 
+team leads, department heads, and managers.
+
+.. _admin-capabilities-unshare-grid:
+
+Unshare Grid View
+-----------------
+
+If this capability is set to *"System"*, the users can unshare grids previously 
+:ref:`shared <admin-capabilities-share-grid>` by themselves. This is usually available to all users who work 
+with grids.
+
+
+.. _admin-capabilities-view-sql:
+
+View SQL Query of a Report/Segment
+----------------------------------
+
+If this capability is set to *"System"*, the users  see the SQL request that is sent to the system for a report/segment.
+
+Usually, this is only granted to system administrators so they can check if a report has been developed correctly.  
+The *"Show SQL Query"* link will appear below the report.
+
+|
+
+.. image:: ./img/roles/sql_show.png
+
+|
+
+
+This setting will only work if it has been enabled within *"System Configuration --> Display Settings --> 
+Report settings*". 
+
+|
+
+.. image:: ./img/roles/sql_setting.png
+
+|
+
+
+.. _admin-capabilities-workflow:
+
+Workflow Manipulations
+----------------------
+
+If this capability is set to *"System"*, users with the role can manage the records, associated with 
+:ref:`workflows <user-guide-workflow-management-basics>`, Otherwise, users may be able to see and edit records, but will 
+not be able to change status of the records within the workflow. 
```

Comment:
Edit: If this capability is set to *"System,"* users with the role can manage the records that are associated with 
:ref:`workflows <user-guide-workflow-management-basics>`. Otherwise, users may be able to see and edit records, but will not be able to change the status of the records within the workflow.